### PR TITLE
Xeora Webserver / Web Framework Addition

### PR DESCRIPTION
**Pull request by freakmaxi**
http://ufal.io
http://jpap.mayadroomajans.com.tr
http://www.mayadroomajans.com.tr/
http://www.resimlerimnette.com/
http://www.mesajkutum.com/
http://www.offlineradio.club/ << it is only used in rest micro service side
http://www.beyinpilim.com/
http://www.jakpark.com/
http://www.bolispirlanta.com/
http://www.xeora.org/

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -9698,6 +9698,23 @@
...
+			],
+			"html": [
+				"<input type=\"hidden\" name=\"_sys_bind_*\" id=\"_sys_bind_*\" />"
+			],
+			"headers": {
+				"Server": "XeoraEngine",
+				"X-Powered-By": "XeoraCube"
+			},
+			"icon": "xeora.png",
+			"script": "/_bi_sps_v*\\.js",
```
Comment (AliasIO) [inline src/apps.json]: Should this be `"/_bi_sps_v.+\\.js"`?

File: src/apps.json [hunk 2]
```diff
@@ -9698,6 +9698,23 @@
...
 			"website": "http://xenforo.com"
 		},
+		"Xeora": {
+			"cats": [
+				"18",
+				"22",
+				"27"
+			],
+			"html": [
+				"<input type=\"hidden\" name=\"_sys_bind_*\" id=\"_sys_bind_*\" />"
```
Comment (AliasIO) [inline src/apps.json]: Should this be `"<input type=\"hidden\" name=\"_sys_bind_*\" id=\"_sys_bind_[^\"]+\" />"`?

File: src/apps.json (hunk 1 above)
Comment (freakmaxi) [reply src/apps.json]: actually, the value is like this `/_bi_sps_v1.0.017.js` in the page. I did first like this `/_bi_sps_v(\d\\.\d\\.\d+)\\.js` but I check other entries and changed like that. Should I changed it like yours?

File: src/apps.json (hunk 2 above)
Comment (freakmaxi) [reply src/apps.json]: and for this, it is the value `<input type="hidden" name="_sys_bind_858321651" id="_sys_bind_858321651" />` in the page. `<input type=\"hidden\" name=\"_sys_bind_(\d+)\" id=\"_sys_bind_(\d+)\" />` was the value and I changed it. I can revert it back to this. what do you say?

File: src/apps.json (hunk 1 above)
Comment (AliasIO) [reply src/apps.json]: It should be like mine. `v*` makes the `v` optional, `v.+` means `v` followed by one or more characters.

File: src/apps.json (hunk 2 above)
Comment (AliasIO) [reply src/apps.json]: Either pattern will work, yours is more specific. You don't need the brackets, `(\d+)` is the same as `\d+` in this case.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**